### PR TITLE
Login frontend 3

### PR DESCRIPTION
**Pull request by patofernandez**
## Summary
The HomeActivity and SignInActivity classes were added without components.
Behavior was given to the buttons and the link of terms and conditions
## Screenshot
image

## Trello
https://trello.com/c/Ksji7j73

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        return R.layout.fragment_home;
+    }
+
+    @Override
+    public void init() {
+    }
+
+    @Override
+    public void onUsernameSaved() {
+        //Intent intent = new Intent(getActivity(), ViewPagerActivity.class);
```
Comment (DemianIac) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.java:25]: Remove this method here to merge. Tell me if it's solved in another branch

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java [hunk 2]
```diff
@@ -60,21 +67,48 @@ public void afterTextChanged(Editable s) {
...
+                    vUserNameInput.setError(getString(R.string.error_formato_invalido));
+                } else if (vPasswordInput.getText().toString().isEmpty()) {
+                    vPasswordInput.setError(getString(R.string.error_campo_incompleto));
+                } else {
+                    guardarUsuario(vUserNameInput.getText().toString());
+                    Toast.makeText(getContext(), "Log In press", Toast.LENGTH_LONG).show();
+                    Intent intent = new Intent(getActivity(), HomeActivity.class);
+                    intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+                    startActivity(intent);
+                    //getPresenter().storeUsername(vUserNameInput.getText().toString());
```
Comment (DemianIac) [inline app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java]: We should remove this.

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java [hunk 3]
```diff
@@ -60,21 +67,48 @@ public void afterTextChanged(Editable s) {
...
-                //getPresenter().storeUsername(vUserNameInput.getText().toString());
+                if (TextUtils.isEmpty(vUserNameInput.getText())) {
+                    vUserNameInput.setError(getString(R.string.error_campo_incompleto));
+                } else if (!validarEmail(vUserNameInput.getText().toString())) {
+                    vUserNameInput.setError(getString(R.string.error_formato_invalido));
+                } else if (vPasswordInput.getText().toString().isEmpty()) {
+                    vPasswordInput.setError(getString(R.string.error_campo_incompleto));
+                } else {
+                    guardarUsuario(vUserNameInput.getText().toString());
+                    Toast.makeText(getContext(), "Log In press", Toast.LENGTH_LONG).show();
```
Comment (DemianIac) [inline app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java]: We should remove this now

File: app/src/main/java/ar/com/wolox/android/example/ui/login/RouteActivity.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    protected void init() {
+        if (readSharedPreferences().equals("")) {
+            Intent intent = new Intent(this, LoginActivity.class);
+            intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+            startActivity(intent);
+        } else {
+            Intent intent = new Intent(this, HomeActivity.class);
+            intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+            startActivity(intent);
+            Toast.makeText(this, "Usuario logueado", Toast.LENGTH_SHORT).show();
```
Comment (DemianIac) [inline app/src/main/java/ar/com/wolox/android/example/ui/login/RouteActivity.java]: We should remove this

File: app/src/main/java/ar/com/wolox/android/example/ui/login/SignInFragment.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        return R.layout.fragment_signin;
+    }
+
+    @Override
+    public void init() {
+    }
+
+    @Override
+    public void onUsernameSaved() {
+        //Intent intent = new Intent(getActivity(), ViewPagerActivity.class);
```
Comment (DemianIac) [inline app/src/main/java/ar/com/wolox/android/example/ui/login/SignInFragment.java:24]: Same here. Tell me if it's solved in other branch

**Review by DemianIac — CHANGES_REQUESTED** [review]

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.java (hunk 1 above)
Comment (patofernandez) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.java:25]: All comments were removed in the login-backend-2 branch

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java (hunk 2 above)
Comment (patofernandez) [reply app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java]: done!

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java (hunk 3 above)
Comment (patofernandez) [reply app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java]: done!

File: app/src/main/java/ar/com/wolox/android/example/ui/login/RouteActivity.java (hunk 4 above)
Comment (patofernandez) [reply app/src/main/java/ar/com/wolox/android/example/ui/login/RouteActivity.java]: done!

File: app/src/main/java/ar/com/wolox/android/example/ui/login/SignInFragment.java (hunk 5 above)
Comment (patofernandez) [reply app/src/main/java/ar/com/wolox/android/example/ui/login/SignInFragment.java:24]: All comments were removed in the login-backend-2 branch

**Review by DemianIac — APPROVED** [review]

**Pull request merged**